lib/memory/index.js

```diff
@@ -103,19 +103,22 @@ module.exports = function MemoryStore(data_set) {
   })
 
 
-  // _sort :: Sort -> [a] -> [a]
-  const _sort = R.curry((sort, list) => {
+  // _sortToFn :: Sort -> ComparatorFn
+  const _sortToFn = ([key, order]) => R.ifElse(
+    R.equals('desc')
```

Comment:
In this case, it'll assume ascend if they don't specify descend. I think if they're using the sort in general, it's implied that they know it'll be sorted.